tests/functional/s3/test_cp_command.py

```diff
@@ -235,6 +235,45 @@ def test_cp_fails_with_utime_errors_but_continues(self):
             _, err, _ = self.run_cmd(cmdline, expected_rc=1)
             self.assertIn('attempting to modify the utime', err)
 
+    def test_recursive_glacier_download_with_force_glacier(self):
+        self.parsed_responses = [
+            {
+                'Contents': [
+                    {'Key': 'foo/bar.txt', 'ContentLength': '100',
+                     'LastModified': '00:00:00Z',
+                     'StorageClass': 'GLACIER',
+                     'Size': 100},
+                ],
+                'CommonPrefixes': []
+            },
+            {'ETag': '"foo-1"', 'Body': six.BytesIO(b'foo')},
+        ]
+        cmdline = '%s s3://bucket/foo %s --recursive --force-glacier' % (
```

Comment:
Should be ``--force-glacier-transfer``. ``--force-glacier`` only works because argparse is weird in that it can autocomplete arguments that were provided that we shortened.